onnxscript/optimizer/constant_folding_test.py

```diff
@@ -3,12 +3,29 @@
 import unittest
 
 import onnx
+import parameterized
 import pytest
 
-from onnxscript import optimizer
+import onnxscript.optimizer as optimizer
+from onnxscript.ir import serde
+from onnxscript.optimizer import _constant_folding, constant_folding
 
 
+@parameterized.parameterized_class(("using_ir",), [(False,), (True,)])
 class FoldConstantsTest(unittest.TestCase):
+    def _fold(self, model: onnx.ModelProto, onnx_shape_inference=False):
+        if self.using_ir:
+            ir_model = serde.deserialize_model(model)
+            _constant_folding.fold_constants(
+                ir_model, onnx_shape_inference=onnx_shape_inference
+            )
+            optimizer.remove_unused_nodes(ir_model)
+            return serde.serialize_model(ir_model)
+        else:
+            constant_folding.fold_constants(model, onnx_shape_inference=onnx_shape_inference)
```

Comment:
Will we replace constant_folding with _constant_folding eventually? If so, _constant_folding should not import stuffs from constant_folding?

Reply:
yes ... they will migrate into this file eventually